src/hotspot/share/opto/loopnode.cpp

```diff
@@ -3917,6 +3917,35 @@ bool PhaseIdealLoop::is_deleteable_safept(Node* sfpt) {
 
 //---------------------------replace_parallel_iv-------------------------------
 // Replace parallel induction variable (parallel to trip counter)
+// This optimization looks for patterns similar to:
+//
+//    int a = init2;
+//    for (int phi = init; phi < limit; phi += stride_con) {
+//      a += stride_con2;
+//    }
+//
+// and transforms it to:
+//
+//    int a = init2
+//    for (int phi = init; phi < limit; phi += stride_con) {
+//      a = init2 + (phi - init) * (stride_con2 / stride_con)
+//    }
+//
+// so that the loop can be eliminated given that `stride_con2 / stride_con` is
```

Comment:
In general, a loop is not eliminated after a parallel iv is found. Can you tweak the comment so it says, that in this particular example the loop can be eliminated,